app-containers/lxd/lxd-4.0.9-r4.ebuild

```
@@ -27,8 +27,11 @@ DEPEND="acct-group/lxd
 	sys-libs/libcap
 	virtual/udev"
 RDEPEND="${DEPEND}
-	net-firewall/ebtables
-	net-firewall/iptables[ipv6(+)?]
+	nftables? ( net-firewall/nftables[json] )
+	!nftables? (
+				 net-firewall/ebtables
+				 net-firewall/iptables[ipv6(+)?]
+			   )
```

We shouldn't really need a use flag for this, since it doesn't have any other function in the ebuild. Something like
```suggestion
	|| (
		net-firewall/nftables[json]
		(
			net-firewall/ebtables
			net-firewall/iptables[ipv6(+)?]
		)
	)
```
should do the trick here. In this case the top selection will be preferred by the package manager, in this example being **nftables**.

---

Thanks for your feedback :) I have implemented your suggestion and rebased the commit.

I chose not to revbump the ebuilds, because users of LXD either work with a functional xtables implementation already or they installed `nftables[json]` manually themselves. Imo, a revbump would cause unnecessary rebuilds downstream. I hope this is correct.